charts/opentelemetry-collector/templates/_helpers.tpl

```diff
@@ -163,6 +163,10 @@ Allow the release namespace to be overridden
   {{- end -}}
 {{- end -}}
 
+{{- define "opentelemetry-collector.retrieveNamespace" -}}
```

Comment:
I see we already have `opentelemetry-collector.namespace`, can we use that instead?

Reply:
I've noticed it before, but I haven't tested it, I'll try to use the existing ones later

Reply:
use `opentelemetry-collector.namespace` works as expected.